androidApp/src/main/java/com/mbta/tid/mbta_app/android/pages/NearbyTransitPage.kt

```diff
@@ -87,12 +89,25 @@ fun NearbyTransitPage(
     val stopDetailsFilter by viewModel.stopDetailsFilter.collectAsState()
     var vehiclesData: List<Vehicle> = subscribeToVehicles(routeDirection = stopDetailsFilter)
 
+    val searchFocusRequester = remember { FocusRequester() }
```

Comment:
question (non-blocking): Why not rememberSaveable? My understanding is that we'd always want to default to using the saveable version

Reply:
I do not think the `FocusRequester` supports `Parcel`, and even if it did, it doesn't contain any data that we need to preserve while the app is in the background, the only important thing is that we use the same instance in `openSearch` and `SearchBarOverlay`, and if it's a different instance after the app comes back from background then that's entirely fine.